src/Microsoft.Extensions.DependencyModel/project.json

```diff
@@ -14,7 +14,7 @@
   },
   "frameworks": {
     "net451": { },
-    "dnxcore50": {
+    "dotnet5.4": {
```

Comment:
Because `dnxcore50` is going away. It doesn't really make sense to have it at all in this case. Eventually it will be replaced by something else, but that TFM will only be for apps anyway. `dotnet5.4` (which will be becoming `netstandard1.3`) is the expected TFM for libraries.